modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -1949,4 +1976,83 @@ public Track xapply(Job job) throws MediaPackageException {
       }
     };
   }
+
+  private MediaPackage checkForCASeries(MediaPackage mp, String seriesAppendName) {
+    //Check for mediapackage id and CA series appendix set
+    if (mp == null || seriesAppendName == null) {
+      logger.debug("No series name provided");
+      return mp;
+    }
+    //Get capture agent name
+    String captureAgentId = null;
+    Catalog[] catalog = mp.getCatalogs(MediaPackageElementFlavor.flavor("dublincore", "episode"));
+    if (catalog.length == 1) {
+      try (InputStream catalogInputStream = workingFileRepository.get(mp.getIdentifier().toString(), catalog[0].getIdentifier())) {
+        DublinCoreCatalog dc = dublinCoreService.load(catalogInputStream);
+        captureAgentId = getCaptureAgent(dc);
+      } catch (Exception e) {
+        logger.warn("Unable to determine capture agent name", e);
```

Comment:
Will this cause a warning being logged every time someone ingests something not from a capture agent?